draft-ietf-quic-manageability.md

```diff
@@ -804,8 +805,9 @@ The QUIC header optionally contains a connection ID which can be used as
 additional entropy beyond the 5-tuple, if needed. The QUIC handshake needs
 to be observed in order to understand whether the connection ID is present and
 what length it has. However, connection IDs may be renegotiated during
-a connection, and this renegotiation is not visible to the path. Keying state
-off the connection ID may therefore cause undetectable and unrecoverable loss
+a connection, and this renegotiation is not visible to the path. Using the
+connection ID as a flow key field for statful treatment of flows may therefore
```

Comment:
```suggestion
connection ID as a flow key field for stateful treatment of flows may therefore
```

Reply:
This sentence isn't quite as cautionary as I think it should be, but I don't have any good suggestion.